### DISCUSSION TITLE
Trying to build for board espressif_esp32s3_box

### DISCUSSION THREAD

**Discussion by PaulskPt** [adafruit/tinyuf2#247] [Q&A] [answered]
During the build I am getting this error:
```Traceback (most recent call last):
  File "../../tools/gen_web_workflow_static.py", line 7, in <module>
    import minify_html
ModuleNotFoundError: No module named 'minify_html'
make: *** [../../supervisor/supervisor.mk:174: build-espressif_esp32s3_box/autogen_web_workflow_static.c] Error 1
```
I did a search for ```minify_html``` but I can't find it either.

Another thing is that the build ends with:
```-- Configuring done
-- Generating done
-- Build files have been written to: /mnt/i/pico/adafruit/circuitpython/ports/espressif/build-espressif_esp32s3_box/esp-idf
```
but no .bin neither .uf2 is created.

Built output states:
 ```App "circuitpython" version: 8.0.0-beta.0-114-g5a053f9ae-dir```

I cloned the latest Circuitpython repo. in /ports/espressif/esp-idf the version is 4.4.1.

**Comment by dhalbert** [comment]
These build problems are for CircuitPython, not tinyuf2. If you are having CircuitPython build problems ask in the #circuitpython-dev channel in discord: https://adafru.it/discord. When you want to make a tinyuf2 build for this board, this is the right place to ask.

**Reply by PaulskPt** [reply]
OK, thank you Dan. I

**Comment by PaulskPt** [comment, marked as answer]
Today, thanks to @hathach who added the board espressif_esp32s3_box,  I successfully built tinyuf2 for board espressif_esp32s3_box.

**Reply by hathach** [reply]
correction: #251 is implemented by @Neradoc , I only reviewed it. I was still trying to find where my S3 box in the drawer :D

**Comment by PaulskPt** [comment]
OK, thanks to both @Neradoc  & @hathach !
